docker/py-dsci-100/.gitconfig

```diff
@@ -0,0 +1,2 @@
+[pull]
+        rebase = true
```

Comment:
Why did we change the pull strategy? I wouldn't normally go for rebase -- any particular reason?

Reply:
I can't find the original slack thread where it was suggested, but no, there's no particular reason. We can use any other strategy. Maybe ff only? 

It's just to suppress this error:
```Hint: You have divergent branches and need to specify how to reconcile them.
Hint: You can do so by running one of the following commands sometime before
Hint: your next pull:
Hint: 
Hint:   git config pull.rebase false  # merge
Hint:   git config pull.rebase true   # rebase
Hint:   git config pull.ff only       # fast-forward only```

Reply:
Ah I see! It's not for *this* repo, it's for copying into the docker image later on. Let's use the default strategy which is `pull.rebase false` (so that what they see in class is the common default in the real world)